common/views/themes/utility-classes.ts

```diff
@@ -400,7 +400,7 @@ export const utilityClasses = css<GlobalStyleProps>`
     &:hover .promo-link__title,
     &:focus .promo-link__title {
       text-decoration: underline;
-      text-decoration-color: ${themeValues.color('green')};
+      text-decoration-color: ${themeValues.color('black')};
```

Comment:
This change will reflect across, so maybe I should change it so it's just on Works, but from my convos with Gareth O, this was the eventual goal.

Reply:
Might as well go for it everywhere (until someone complains).